nautobot/extras/api/relationships.py

```diff
@@ -267,13 +307,17 @@ class RelationshipModelSerializerMixin(ValidatedModelSerializer):
     def create(self, validated_data):
         relationships = validated_data.pop("relationships", {})
         instance = super().create(validated_data)
+        validate_relationships(self.context["request"], instance, relationships)
```

Comment:
In the case of an error here, does the instance still get created since you're calling `super().create(validated_data)` before the `validate_relationships()` check? Or does raising an exception cause it to not get created after all?